components/http/okHttp/src/main/java/com/microsoft/kiota/http/KiotaClientFactory.java

```diff
@@ -92,11 +102,60 @@ private KiotaClientFactory() {}
         };
     }
 
+    /**
+     * Creates the default interceptors for the client.
+     * @return an array of interceptors.
+     */
+    @Nonnull public static Interceptor[] createDefaultInterceptors( @Nullable Interceptor[] interceptors) {
```

Comment:
what's the value of this method? couldn't we rely on the existing one or the one that accepts options instead to avoid a sprawling public API surface?

Reply:
I'm not sure I understand the comment here? We already had a createDefaultInterceptors method (no parameters) and we're not taking it away.
We're introducing an overload which accepts a list of options.
How is this method, which accept an array of options preventing any source breaking change?
I feel like we could achieve the same result without this specific method.

Reply:
@baywet  just want to clarify we dont want this overload 
`  @Nonnull public static Interceptor[] createDefaultInterceptors( @Nullable Interceptor[] interceptors) ` is this accurate ?

Reply:
that's my suggestion yes, it doesn't add any value unless I missed something

Reply:
thanks updated the code sorry for confusion